### ISSUE TITLE
Error when opening PowerShell ISE after install (NullReference in PowerShellIse.OnLoad)

### ISSUE THREAD

**Issue by zenima** [SitecorePowerShell/Console#426]
Hi!

I've just installed **PowerShell ISE 3.1** on a (relatively) new **Sitecore 8 (rev 150427)** instance. I had no previous versions of PowerShell on this instance before.
The install went solid, I had no errors / warning / overwritten items. After I restarted the instance I tried to launch the PowerShell ISE, but I received the following error:

```
[NullReferenceException: Object reference not set to an instance of an object.]
   Cognifide.PowerShell.Client.Applications.PowerShellIse.OnLoad(EventArgs e) +1397

[TargetInvocationException: Exception has been thrown by the target of an invocation.]
   System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor) +0
   System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments) +76
   System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture) +211
   System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters) +35
   Sitecore.Web.UI.Sheer.ClientPage.OnLoad(EventArgs e) +337
   System.Web.UI.Control.LoadRecursive() +71
   System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint) +3178
```

I haven't found any module related entries in the log file before or after this. Other parts of the module work, at least those I tried (listing PowerShell commands, etc)

Is this bug, or there might be a faulty configuration in my environment? Any ideas what should I check?

Thank you!

**Comment by AdamNaj** [comment]
Potential causes could be as described in issues: 
- #394 - Missing Home item (fixed in 3.2 - interim workaround found in the issue conversation)
- #404 - another potential cause is that the package didn't install in full as described in the issue because Mongo is not fully setup and analytics is enabled.

Hope this helps resolve the issue.

**Comment by zenima** [comment]
Thanks for the quick response! Adding the proper contentStartPath to the shell site as in #394 solved it.
